metrics/sink.go

```diff
@@ -16,9 +16,8 @@ var (
 )
 
 type Sink interface {
-	Add(s Sample)                              // Add a sample to the sink.
-	Format(t time.Duration) map[string]float64 // Data for thresholds.
-	IsEmpty() bool                             // Check if the Sink is empty.
+	Add(s Sample)  // Add a sample to the sink.
```

Comment:
Can we use this opportunity to have standard godoc comments for those methods? 😸